core/src/main/scala/org/apache/spark/status/api/v1/SimpleDateParam.scala

```diff
@@ -39,17 +49,3 @@ private[v1] class SimpleDateParam(val originalValue: String) {
     )
   }
 }
-
-private[v1] object SimpleDateParam {
-
-  val formats: Seq[SimpleDateFormat] = {
```

Comment:
why not just make this a `def makeFormats` or something? The old code is quite readable.